src/types/global.d.ts

```diff
@@ -106,17 +106,21 @@ declare global {
 }
 
 declare module 'chromecast-caf-receiver/cast.framework.messages' {
-    interface MediaInformationCustomData {
-        audioStreamIndex: number | null;
-        canClientSeek: boolean;
-        canSeek: boolean;
-        itemId: string | undefined;
-        liveStreamId: string | null;
-        mediaSourceId: string | null;
-        playMethod: 'DirectStream' | 'Transcode';
-        playSessionId: string;
-        runtimeTicks: number | null;
-        startPositionTicks: number;
-        subtitleStreamIndex: number | null;
-    }
+    // eslint-disable-next-line @typescript-eslint/no-empty-object-type
```

Comment:
I can't see any way to avoid this.

As an alternative the rule can be configured to ignore empty interfaces that extend from a single interface, but that might not be appropriate.